src/components/Tooltip/Tooltip.module.css

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2023 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.tooltip {
+  font: var(--cpd-font-body-xs-medium);
+  padding: var(--cpd-space-1-5x) var(--cpd-space-3x);
+  background: var(--cpd-color-alpha-gray-1400);
```

Comment:
`--cpd-color-alpha-gray-1400` out of interest, how does the light ↔ dark theme switch work? Does it replace `--cpd-color-alpha-gray-1400`?

Reply:
The colour scales are inverted. `--cpd-color-alpha-gray-1400` is dark in the ligtht theme and vice versa.